presto-main/src/test/java/com/facebook/presto/cost/SymbolStatsAssertion.java

```diff
@@ -106,7 +106,8 @@ public SymbolStatsAssertion dataSizeUnknown()
         return this;
     }
 
-    public SymbolStatsAssertion isEqualTo(SymbolStatsEstimate expected) {
+    public SymbolStatsAssertion isEqualTo(SymbolStatsEstimate expected)
+    {
```

Comment:
move it to previous commit

Reply:
i inserted a fixup between the existing commits. Sorry for messing up the GH history